pages/administratorWorkspaces/administratorWorkspaces.html.js

```diff
@@ -0,0 +1,190 @@
+                  <div class="list-group-item">
+                    <div class="d-flex flex-row flex-wrap align-items-center">
+                      <div class="d-flex align-items-center mr-auto pr-2 text-monospace">
+                        <a
+                          href="workspaces-${workspaceHost.id}"
+                          class="mr-2"
+                          data-target="#workspaces-${workspaceHost.id}"
+                          data-toggle="collapse"
+                          aria-expanded="false"
+                          aria-controls="workspaces-${workspaceHost.id}"
+                          >${workspaceHost.hostname}</a
+                        >
+                        ${instanceId
+                          ? html`<span class="text-muted mr-2">(${instanceId})</span>`
+                          : null}
+                        ${WorkspaceHostState({ state: workspaceHost.state })}
+                        <span class="badge badge-secondary">${workspaceHost.time_in_state}</span>
+                      </div>
+                      ${Capacity({
+                        total: workspaceLoadHostCapacity,
+                        current: workspaceHost.workspaces.length,
+                      })}
+                    </div>
+                    <div id="workspaces-${workspaceHost.id}" class="collapse">
+                      <div class="list-group list-group my-2">
+                        ${workspaceHost.workspaces.map((workspace) => {
+                          return html`
+                            <div class="list-group-item d-flex align-items-center">
+                              <span class="mr-2" style="font-variant-numeric: tabular-nums;">
+                                ${workspace.id}
+                              </span>
+                              ${WorkspaceState({ state: workspace.state })}
+                              <span class="badge badge-secondary">
+                                ${workspace.time_in_state}
+                              </span>
+                            </div>
+                          `;
+                        })}
+                      </div>
+                    </div>
+                  </div>
+                `;
+              })}
+            </div>
+          </div>
+        </div>
+      </body>
+    </html>
+  `.toString();
+}
+
+/**
+ * @typedef {Object} CapacityProps
+ * @property {number} total
+ * @property {number} current
+ */
+
+/**
+ * @param {CapacityProps} props
+ * @returns {import('@prairielearn/html').HtmlSafeString}
+ */
+function Capacity({ total, current }) {
+  const capacity = (current / total) * 100;
+  return html`
+    <div class="d-flex flex-row align-items-center">
+      <div class="text-muted mr-2" style="font-variant-numeric: tabular-nums;">
+        ${current} / ${total}
+      </div>
+      <div class="progress flex-grow-1" style="width: 100px">
+        <div class="progress-bar" style="width: ${capacity}%"></div>
```

Comment:
```suggestion
        <div class="progress-bar" style="width: ${capacity}%" aria-valuenow="${capacity}" aria-valuemin="0" aria-valuemax="100" role="progressbar"></div>
```

For screen reader accessibility

Reply:
Wouldn't this be more accurately described as a [meter](https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/Roles/meter_role)?

Reply:
I added meter attributes in 902c415.

Reply:
Ah good point! I had just copied from the bootstrap docs, but `meter` fits better here.